client/listingdetails.css

```diff
@@ -0,0 +1,78 @@
+.container {
+    display: flex;
+    height: 128px;
+    align-content: space-around;
+}
+
+.header {
+  margin: 20px;
+  height: 60px;
+}
+
+.item {
+  margin: 20px;
+}
+
+.list {
+    height: 400px;
+    margin-top: 16px;
+    margin-bottom: 16px;
+    word-wrap: break-word !important;
```

Comment:
As far as I understand, !important is generally something you want to avoid. I'm not quite sure on the details yet, but I believe there are ways to set up your css so as to avoid using '!important'.
https://www.quora.com/Is-it-wrong-to-use-important-in-CSS